src/OneExplorer/OneExplorer.ts

```diff
@@ -671,13 +701,32 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<Node> {
         OneStorage.reset();
         // Reset the root in order to build from scratch (at OneTreeDataProvider.getTree)
         this._tree = undefined;
+        // Update selection info
+        this.refreshSelection();
       }
       this._onDidChangeTreeData.fire(undefined);
     } else {
       this._onDidChangeTreeData.fire(node);
     }
   }
 
+  /**
+   * Refresh hasSelectedCfg parameter
+   */
+  refreshSelection(): void {
```

Comment:
How about 'refreshCfgSelection()' for the name? By the name, reader might misunderstand that the function refreshes general selection of nodes.

Reply:
Ahhh... Ok. I see.

Reply:
Thanks!